wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - medals
+  - medal hunting
+---
+
+# osu! Medal Hunters Discord server
+
+::: Infobox
+
+<!-- lint ignore heading-increment -->
+
+#### osu! Medal Hunters
+
+![Server logo](img/logo.png "The Medal Hunter Discord server logo, designed by Tanza3D")
+
+|  |  |
+| :-- | :-- |
+| Invite link | <https://discord.gg/osumedalhunters> |
+| Website | <https://osekai.net/> |
+
+:::
+
+*Notice: While working together in order to find Hush-Hush medals is encouraged, finding them by yourself is as well. Be advised that the website and server contain solutions to Hush-Hush medals.*
+
+**osu! Medal Hunters** is a the main [Discord](https://discord.com) server used by the medal hunting community, where people discuss [medals](/wiki/Medals) and [medal solutions](/wiki/Medals/Unlock_requirements).
+
+The server is run by the same team that develops and maintains the [Osekai website](https://osekai.net/), which is a hub for alternative rankings, medal solutions, archived osu! client versions, and a [profile badge](/wiki/Community/Profile_badge) database.
+
+A lot of medal hunters work together in order to find solutions for each new Hush-Hush medal added into the game. Every time new medals get released, it is announced by one of the Osekai staff members, threads for each medal are created, and people work together to hunt down the solutions.
+
+## Staff
+
+| Role | User (Discord handle) |
+| :-- | :-- |
+| Server owner | [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
+| Osekai developer & server admin | ::{ flag=UK }:: [Tanza3D](https://osu.ppy.sh/users/10379965) (Hubz#6283, Tanza#8733), ::{ flag=DE }:: [mulraf](https://osu.ppy.sh/users/1309242) (mulraf#1087) |
+| Osekai community manager & server admin | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151), ::{ flag=UK }:: [chromb](https://osu.ppy.sh/users/10238680) (chromb#4528), ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
```

Comment:
```suggestion
| Osekai community manager & server admin | ::{ flag=UA }:: [MegaMix_Craft](https://osu.ppy.sh/users/18152711) (MegaMix_Craft#1151), ::{ flag=GB }:: [chromb](https://osu.ppy.sh/users/10238680) (chromb#4528), ::{ flag=CA }:: [Remyria](https://osu.ppy.sh/users/1699875) (Remyria#0001) |
```

Reply:
Hey, @Hiviexd, I'm not from GB, so country code is actually correct for this one!